packages/apps/frontera/src/routes/organizations/src/components/Search/Search.tsx

```diff
@@ -182,10 +191,18 @@ export const Search = observer(() => {
       ? 'e.g. Isabella Evans'
       : 'e.g. CustomerOS...';
 
+  const handleOpenICPFlow = () => {
```

Comment:
let's change name to 'handleToggleIcpFlow'

Reply:
i will rename it as handleToogleFlow ( to be more generic because will open also the persona flow)